packages/typespec-test/test/confidentialLedger/generated/openapi/2022-05-13/openapi.json

```diff
@@ -206,7 +206,8 @@
           }
         },
         "x-ms-pageable": {
-          "nextLinkName": "nextLink"
+          "nextLinkName": "nextLink",
```

Comment:
could you check relevant pr change?

Reply:
pr link: https://github.com/Azure/typespec-azure/pull/1449/files